cmd/deploy/endpoints.go

```diff
@@ -189,14 +189,16 @@ func (eg *EndpointGetter) getEndpoints(ctx context.Context, opts *EndpointsOptio
 		return nil, err
 	}
 
-	externalEps, err := eg.endpointControl.List(ctx, opts.Namespace, labelSelector)
-	if err != nil {
-		return nil, err
-	}
+	if okteto.Context().IsOkteto {
```

Comment:
is this part required for the change? if not can we move it to additional PR, the diff coverage is being affected by this change, which is not tested, and reading the issue i think it might be an additional change not required for the messagE?

Reply:
oh! I think I've pushed that commit by wrong in this branch. :hushed: Good catch Tere! I'll revert that.